attributes/default.rb

```diff
@@ -474,9 +474,12 @@
 default['hops']['docker']['registry']['port']         = 4443
 default['hops']['docker']['registry']['download_url'] = "#{node['download_url']}/kube/docker-images/#{node['hops']['docker_img_version']}/registry_image.tar"
 default['hops']['docker']['pkg']['download_url']['centos'] ="#{node['download_url']}/docker/#{node['hops']['docker_version']['centos']}/rhel"
-default['hops']['nvidia_pkgs']['download_url']        ="#{node['download_url']}/kube/nvidia"
+default['hops']['nvidia_pkgs']['download_url']        = "#{node['download_url']}/kube/nvidia"
 
 #XAttrs
 default['hops']['xattrs']['enabled']                  = "true"
 default['hops']['xattrs']['max-xattrs-per-inode']     = 32
 default['hops']['xattrs']['max-xattr-size']           = 1039755
+
+#AWS
+default['hops']['redshift_url']                       = "https://s3.amazonaws.com/redshift-downloads/drivers/jdbc/1.2.47.1071/RedshiftJDBC42-no-awssdk-1.2.47.1071.jar"
```

Comment:
The fact that you are on AWS doesn't mean that you have access to the outside world. Plenty of enterprises lock down their accounts.

Reply:
I can put the jars on our server then.